notebooks/regression/exoplanets.md

```diff
@@ -0,0 +1,373 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.6
+  kernelspec:
+    display_name: mapie_local
+    language: python
+    name: mapie_local
+---
+
+# Estimating the uncertainties of exoplanet masses
+
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/scikit-learn-contrib/MAPIE/blob/master/notebooks/regression/exoplanets.ipynb)
+
+
+
+In this notebook, we will quantify the uncertainty of exoplanet masses predicted by several machine learning models based on the exoplanet properties. To this aim, we will use the exoplanet dataset downloaded from the [NASA Exoplanet Archive](https://exoplanetarchive.ipac.caltech.edu/) and estimate the prediction intervals using the methods implemented in MAPIE.
+
+```python
+install_mapie = True
+if install_mapie:
+    !pip install mapie
+```
+
+```python
+import warnings
+from typing_extensions import TypedDict
+from typing import Union
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+from sklearn.compose import ColumnTransformer
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.impute import SimpleImputer
+from sklearn.linear_model import LinearRegression
+from sklearn.model_selection import train_test_split, KFold
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import (
+    RobustScaler,
+    OneHotEncoder,
+    OrdinalEncoder,
+    PolynomialFeatures
+)
+from mapie.metrics import regression_coverage_score
+from mapie.regression import MapieRegressor
+from mapie.subsample import Subsample
+warnings.filterwarnings("ignore")
```

Comment:
I am a little maniac obsessive so could you skip a line between native-python and local pacakges ?
And it would be wonderful if the imports are done in à alphabetical order (first the "from" imports then the "import ... as ..."

Reply:
Yes you are ! Done.